src/app/+risk-calc-advanced/containers/trade-form/trade-form.component.html

```diff
@@ -0,0 +1,54 @@
+<form [formGroup]="form" (change)="onChange()">
+  <div class="trade-form mat-elevation-z8">
+    <h4>Common</h4>
+
+    <app-common-panel [group]="form.get('commonPanel')"
+                      (dataChange)="onChange()"
+    ></app-common-panel>
+  </div>
+
+  <div class="trade-form mat-elevation-z8">
+    <h4>Entries</h4>
+
+    <app-entry-price formName="entries"
+                     [group]="form.get('entries')"
+                     [calculationData]="[{Vq:1, Vb:2}, {Vq:1, Vb:2}]"
+                     (addOrderItemAbove)="addOrderItemAbove('entries', $event)"
+                     (addOrderItemBelow)="addOrderItemBelow('entries', $event)"
+                     (removeOrderItem)="removeOrderItem('entries', $event)"
+                     (setOrderItemPercentage)="onSetOrderItemPercentage($event)"
+                     (equalizePercentage)="equalizePercentage('entries')"
+                     (dataChange)="onChange()"
+    ></app-entry-price>
+  </div>
+
+  <div class="trade-form mat-elevation-z8">
+    <h4>Stop Losses</h4>
+
+    <app-stop-loss formName="stops"
+                   [group]="form.get('stops')"
+                   [calculationData]="[{Vq:1, Vb:2}, {Vq:1, Vb:2}]"
+                   (addOrderItemAbove)="addOrderItemAbove('stops', $event)"
```

Comment:
Let's pass the instance of the `FormGroup` here instead of its name. Example:
`(addOrderItemAbove)="addOrderItemAbove(stopOrdersForm, $event)"`